Test/RivetTest/Functions/Start-RivetTest.ps1

```diff
@@ -4,11 +4,13 @@ function Start-RivetTest
     [CmdletBinding()]
     param(
         # Optional Parameter to specify a plugin Path
-        [String[]]$PluginPath,
+        [String[]] $PluginPath,
 
-        [String[]]$IgnoredDatabase,
+        [String[]] $IgnoredDatabase,
 
-        [String[]]$DatabaseName = $RTDatabaseName
+        [String[]] $DatabaseName = $RTDatabaseName,

```

Comment:
Rename this to `PhysicalDatabase` (with a `[Alias('DatabaseName')]` alias unless you feel like updating all the usages).